lib/util/cache/repository/types.ts

```diff
@@ -42,6 +42,12 @@ export interface OnboardingBranchCache {
   configFileParsed?: string;
 }
 
+export interface ReconfigureBranchCache {
+  reconfigureBranchSha: string;
+  configFileName: string | null;
```

Comment:
How/why is this field reused? The user should be able to change file name between commits anyway. I think it can be removed?